portfolio/src/main/webapp/college.html

```diff
@@ -0,0 +1,3 @@
+<!--#Do a timeline of important events
+#start with after highschool graduction
+#conclude with start of STEP intern ship-->
```

Comment:
This newline thing is super annoying. No need to fix now though. This isn't blocking.